helm-deploy-test/tasks/cf-teardown.sh

```diff
@@ -11,11 +11,9 @@ UAA_NAMESPACE=uaa
 set +o allexport
 
 for namespace in "$CF_NAMESPACE" "$UAA_NAMESPACE" ; do
-    for name in $(helm list --deployed --short --namespace "${namespace}") ; do
-        helm delete "${name}" || true
-    done
-    while kubectl get namespace "${namespace}" ; do
-        kubectl delete namespace "${namespace}" || true
+    kubectl delete namespace "${namespace}" || true
+    while [[ -n $(helm list --short --all ${namespace}) ]]; do
         sleep 10
+        helm delete --purge ${namespace};
```

Comment:
Really tiny nit: do we need the semicolon here? Also, quoting (thought that should be safe)

I believe we had the `|| true` before because it would sometimes fail erroneously (because the machine was too slow and connection to the database times out). Is that no longer a concern?

Do we not need to care about `kubectl delete` failing intermittently?

(I don't actually care about the nits, just the other stuff)

Reply:
These are great nits.. not sure how/why I added the semicolon, and the `||:` is probably still a good idea